tests/playbooks/tests_team_exists.yml

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    network_connections:
+      # Specify the team profile
+      - name: team0
+        state: up
+        type: team
+        interface_name: team0
+        # ip configuration (optional)
+        ip:
+          address:
+            - "192.0.2.24/24"
+            - "2001:db8::23/64"
+
+  tasks:
+      - name: Get NetworkManager-team RPM version
+        command:
+          cmd: rpm -qa --qf '%{name}-%{version}-%{release}\n' NetworkManager-team
+          warn: false
+        register: __rpm_q_NetworkManager
```

Comment:
This is gathering name-version-release but not checking that the package exists.